tests/run/test_fstring.pyx

```diff
@@ -1122,21 +1122,22 @@ non-important content
         self.assertEqual(cy_eval('f"\\\n"'), '')
         self.assertEqual(cy_eval('f"\\\r"'), '')
 
-    """
-    def __test_debug_conversion(self):
+    def test_debug_conversion(self):
         x = 'A string'
         self.assertEqual(f'{x=}', 'x=' + repr(x))
         self.assertEqual(f'{x =}', 'x =' + repr(x))
         self.assertEqual(f'{x=!s}', 'x=' + str(x))
         self.assertEqual(f'{x=!r}', 'x=' + repr(x))
-        self.assertEqual(f'{x=!a}', 'x=' + ascii(x))
+        if not IS_PY2: # ASCII formatting only works on Python 3+
```

Comment:
I the comment is probably wrong - i think `!a` in an f-string works on Cython on Py2, but it's the `ascii` builtin that's missing on Py2 (and thus it's hard to test). Skipping the test may still be reasonable though.

Reply:
Re-worded :)